ktor-server/ktor-server-tests/jvmAndNix/test/io/ktor/tests/server/routing/RoutingResolveTest.kt

```diff
@@ -452,8 +453,8 @@ class RoutingResolveTest {
             }
         }
 
-        on("resolving /foo") {
-            val result = resolve(root, "/foo")
+        on("resolving /foo/") {
```

Comment:
Could you tell me why the old test has been changed? we need to make sure that we don't break any old cases or at least document the breaking change

Reply:
I forgot to roll back this change in the test actually, the behaviour in this case will remain the same.  I had originally changed it but decided it would be too big of a breaking change to carry it forward.  It was originally a fix the first case in the ticket:

Expected:

    /api/subtree - do not match (according to [KTOR-372](https://youtrack.jetbrains.com/issue/KTOR-372/Ktor-should-resolve-trailing-slash-as-a-different-URL) logic)

Current behavior:

    /api/subtree - match
    
But as it turns out we rely quite a bit on this behaviour, especially for serving static files and the like.  I think if we want to change this particular one it will need to be a separate ticket with some configuration item.